### PR TITLE
Add deleteE command

### LINKED ISSUE #96: Add deleteE command

### PR DESCRIPTION
**Pull request by vevek**
Closes #96

### CODE REVIEW

**Comment by vevek** [comment]
Ready for review. Will add tests soon after merge.

File: src/main/java/seedu/address/logic/commands/DeleteEventCommand.java [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+            + PREFIX_REMOVE_TASK_INDEX + "EVENT_INDEX \n"
+            + "Example: " + COMMAND_WORD + " 1" + " "
+            + PREFIX_REMOVE_TASK_INDEX + " 2";
+
+    private final Index projectIndex;
+    private final Index targetEventIndex;
+
+    /**
+     * Creates a DeleteEventCommand to delete the specified {@code Event} from {@code Project}.
+     * @param projectIndex Index of project that {@code Event} is to be deleted from.
```
Comment (samuelfangjw) [inline src/main/java/seedu/address/logic/commands/DeleteEventCommand.java:33]: blank line

**Review by samuelfangjw — DISMISSED** [review]
LGTM

**Review by samuelfangjw — APPROVED** [review]
LGTM

**Pull request merged**